src/ctapipe/tools/tests/test_calculate_pixel_stats.py

```diff
@@ -100,3 +100,18 @@ def test_tool_config_error(tmp_path, dl1_image_file):
             cwd=tmp_path,
             raises=True,
         )
+    # Check if ToolConfigurationError is raised
+    # when the chunk size is larger than the number of events in the input file
+    with pytest.raises(ToolConfigurationError):
```

Comment:
the tool will raise `ToolConfigurationError` in a few situations. Please test all of them and use regexp matching to check whether a correct error message is displayed, e.g. 

```suggestion
    with pytest.raises(ToolConfigurationError, match="Change --StatisticsAggregator.chunk_size")):
        ...
```

Reply:
Good point. Actually, I found that the first raise pytest wasn't raised on the 'column' check but on the 'chunk_size' check which comes before in the tool.